apps/backend-sls/test/integration.test.ts

```diff
@@ -1,13 +1,241 @@
+import { PrismaClient } from '@klicker-uzh/prisma'
+import Redis from 'ioredis'
 import request from 'supertest'
+import prepareApp from '../GraphQL/app'
 
-import app from '../src/app'
+// TODO: switch to ioredis-mock
+// jest.mock('ioredis', () => Redis)
 
-describe('GraphQL API', () => {
-  it('works', async () => {
+process.env.NODE_ENV = 'development'
```

Comment:
If this is not set, GraphQL Yoga will swallow errors and only display "internal error" or similar for anything that breaks inside a service/resolver, which caused me to debug for a few hours unnecessarily.